minutes/2019-11-12.md

```diff
@@ -0,0 +1,65 @@
+# Jakarta EE Platform Call #2
```

Comment:
I'm not sure it's going to be helpful to number each of the calls.  Isn't the date good enough?

Also, I'm not sure we decided whether the meeting minutes need to be approved by an official vote of the committers, in which case this ought to be labeled DRAFT in the title until it's approved.

Reply:
I agree with the date instead of numbering each meeting.

Since these are just development team minutes, I don't think we need any special voting or approval process.  Why not just require two approvals and maybe a 48 hour or 72 hour window?  After that, we can just merge.  Simple process without requiring a separate Draft vs Final designation.  When it's merged, it's final.